java-manta-client/src/main/java/com/joyent/manta/config/SettableConfigContext.java

```diff
@@ -291,6 +299,14 @@ static void setAttributeFromContext(final String name,
             case EnvVarConfigContext.MANTA_TCP_SOCKET_TIMEOUT_ENV_KEY:
                 config.setTcpSocketTimeout(MantaUtils.parseIntegerOrNull(value));
                 break;
+            case MapConfigContext.MANTA_UPLOAD_BUFFER_SIZE_KEY:
+            case EnvVarConfigContext.MANTA_UPLOAD_BUFFER_SIZE_ENV_KEY:
+                config.setUploadBufferSize(MantaUtils.parseIntegerOrNull(value));
```

Comment:
This option was missing from this `switch` but is unrelated to `MANTA_SKIP_DIRECTORY_DEPTH`